### DISCUSSION TITLE
World map chart

### DISCUSSION THREAD

**Discussion by TheophileDiot** [chartbrew/chartbrew#111] [Ideas]
# Context
I was testing the project and had the idea that it would be cool if we could select a chart that shows a world map that shows data according to the location of the data (*given by the alpha_2 of the country, geoname ID or the ISO code*).

> I don't know what do you think about this feature but I think it would be very usefull.

Examples: [Choropleth](https://leafletjs.com/SlavaUkraini/examples/choropleth/), [GDP by country visualization](https://jvectormap.com/examples/world-gdp/)

**Comment by razvanilin** [comment]
Just saw this because GH didn't notify me about the discussion 🤦

I really like the idea! I looked at LeafletJS and the library is awesome. I always wanted to add map support to Chartbrew, but at the moment it's very hard to prioritize the features. I will play around with this soon and measure the complexity of the task. I'll open a PR soon so it's easier to keep track of the implementation
